tardis/montecarlo/tests/test_cmontecarlo.py

```diff
@@ -119,6 +119,9 @@ def model():
         line_lists_j_blues=(c_double * 2)(*([1.e-10] * 2)),
         line_lists_j_blues_nd=0,
 
+        line_lists_Edotlu=(c_double * 3)(1e-10,1e-10,1.0), # Init to an explicit array 
```

Comment:
@Tobychev Please replace this initialization with
```
(c_couble * 3)(*[1.e-10, 1.e-10, 1.0])
```
I think this might be causing the problem. @yeganer what do you think about it ?